packages/button/stories/DefaultButton.stories.tsx

```diff
@@ -242,6 +242,46 @@ storiesOf("Buttons/Default", module)
       }
     }
   )
+  .add(
+    "PrimaryButton used as a link",
```

Comment:
I don't think this story is necessary. If we want to show multiple button appearances being used as a link, I think we should do it in one story

Reply:
The problem is that PrimaryButton used as a link didn't have the expected color.

Reply:
I understand why you're showing the PrimaryButton with a `url` prop, but I think we should just use one story to show different button styles with `url`.

Reply:
Okay, I agree.

Reply:
I pushed a commit to take care of Storybook updates.